### PR TITLE
[cherry-pick] update doc

### PR DESCRIPTION
**Pull request by jerrywgz**
cherry-pick https://github.com/PaddlePaddle/PaddleDetection/pull/4664

### CODE REVIEW

File: docs/MODEL_ZOO_cn.md [hunk 1]
```diff
@@ -93,3 +93,31 @@ Paddle提供基于ImageNet的骨架网络预训练模型。所有预训练模型
...
+
+### DeepSort
+
+请参考[DeepSort](https://github.com/PaddlePaddle/PaddleDetection/tree/develop/configs/mot/deepsort)
+
+### JDE
+
+请参考[JDE](https://github.com/PaddlePaddle/PaddleDetection/tree/develop/configs/mot/jde)
+
+### fairmot
```
Comment (qingqing01) [inline docs/MODEL_ZOO_cn.md]: fairmot -> FairMOT

File: docs/MODEL_ZOO_en.md [hunk 2]
```diff
@@ -92,3 +92,31 @@ Please refer to[PicoDet](https://github.com/PaddlePaddle/PaddleDetection/tree/de
...
+
+### DeepSort
+
+Please refer to [DeepSort](https://github.com/PaddlePaddle/PaddleDetection/tree/develop/configs/mot/deepsort)
+
+### JDE
+
+Please refer to [JDE](https://github.com/PaddlePaddle/PaddleDetection/tree/develop/configs/mot/jde)
+
+### fairmot
```
Comment (qingqing01) [inline docs/MODEL_ZOO_en.md]: 同上

File: docs/MODEL_ZOO_cn.md (hunk 1 above)
Comment (jerrywgz) [reply docs/MODEL_ZOO_cn.md]: done, thx

File: docs/MODEL_ZOO_en.md (hunk 2 above)
Comment (jerrywgz) [reply docs/MODEL_ZOO_en.md]: done, thx

**Review by nemonameless — DISMISSED** [review]

**Review by nemonameless — APPROVED** [review]

**Review by heavengate — APPROVED** [review]

**Pull request merged**